mindmeld/active_learning/classifiers.py

```diff
@@ -168,9 +172,70 @@ def _validate_class_level_statistic(class_level_statistic):
             )
         return class_level_statistic
 
+    @staticmethod
+    def _get_entity_probs(
+        classifier: Classifier,
+        queries: ProcessedQueryList,
+        tag_to_id: Dict,
+    ):
+        """Get the probability distribution for a query across domains or intents.
```

Comment:
Let's update the description here to explicitly explain how the entity probabilities are calculates and what the rows/columns of the final output represent. An small example here would help